specification/apicenter/DataApi/models/definition.tsp

```diff
@@ -0,0 +1,28 @@
+namespace ApiCenter.DataApi.Models;
+
+@doc("API definition")
+model ApiDefinition {
+  @doc("The resource name.")
+  @maxLength(50)
+  name: string;
```

Comment:
Is this a key, part of the URL resource? Should have the `@key` decorator.

Reply:
Fixed in latest revision.